roles/backend_setup/tasks/main.yml

```diff
@@ -1,23 +1,23 @@
 ---
-# A vdo is created an will be used for setting up bricks instead of physical
-# volumes
 # Create a logical volume thinpool when thinpoolname is given
-- name: Install lvm tools for setting up backend
+- name: Change to Install lvm tools for debian systems.
   package:
     name: thin-provisioning-tools
     state: present
     when: ansible_distribution == "Debian" or ansible_distribution == "Ubuntu"
```

Comment:
@papanito this is giving me an error when I test it, because you have not aligned the when: statement with package.